src/AppBundle/Committee/CommitteeCreationCommand.php

```diff
@@ -80,12 +67,13 @@ class CommitteeCreationCommand
      */
     public $acceptContactingTerms;
 
-    public function __construct(Adherent $adherent)
+    public function __construct(Adherent $adherent, Address $address = null)
     {
         $this->adherent = $adherent;
         $this->country = 'FR';
         $this->acceptConfidentialityTerms = false;
         $this->acceptContactingTerms = false;
+        $this->address = $address ?? new Address();
```

Comment:
`$address ?: new Address()`, no need to test if the variable is set, only if it's not null.